src/pool-cl/libraries/CLPool.sol

```diff
@@ -70,6 +76,7 @@ library CLPool {
         mapping(int24 => Tick.Info) ticks;
         mapping(int16 => uint256) tickBitmap;
         mapping(bytes32 => CLPosition.Info) positions;
+        PositionKey[] positionKeys;
```

Comment:
Btw, did u guys ever hear a attack vector that is based on unlimited sized array. Given that:
1. Pushing element into array doesn't cost much in our case (consider addLiquidity 1 to a new range or through new addr)
2. Array elements are laid linearly in evm storage which means the slot position of a given element can be calculated beforehand

If a new element's slot position hits exactly a existing contract variable for example `address _owner`, hacker could manipulate that value through constantly push array into that position

I cant remember where did i see it and not sure if this is feasible. Just thought it is worth to point out as reference